hazelcast/src/main/java/com/hazelcast/client/impl/ClientHeartbeatMonitor.java

```diff
@@ -86,14 +86,14 @@ private void monitor(String memberUuid, ClientEndpointImpl clientEndpoint) {
         }
 
         final Connection connection = clientEndpoint.getConnection();
-        final long lastTimePackageReceived = connection.lastReadTimeMillis();
+        final long lastTimePacketReceived = connection.lastReadTimeMillis();
         final long timeoutInMillis = SECONDS.toMillis(heartbeatTimeoutSeconds);
         final long currentTimeMillis = Clock.currentTimeMillis();
-        if (lastTimePackageReceived + timeoutInMillis < currentTimeMillis) {
+        if (lastTimePacketReceived + timeoutInMillis < currentTimeMillis) {
             if (memberUuid.equals(clientEndpoint.getPrincipal().getOwnerUuid())) {
                 String message = "Client heartbeat is timed out, closing connection to " + connection
-                        + ". Now: " + timeToString(lastTimePackageReceived)
-                        + ". LastTimePacketReceived: " + timeToString(lastTimePackageReceived);
+                        + ". Now: " + timeToString(Clock.currentTimeMillis())
```

Comment:
This may not be important but just for my curiosity what is the true `now` here? Is it `Clock.currentTimeMillis()` or previously calculated `currentTimeMillis `?